File: Kattis/C++/1_Easy/bijele.cpp
#include <iostream>
#include <vector>
using namespace std;

int main()
{
    vector<int> pieceset = {1,1,2,2,2,8};

    vector<int> input(6);

    vector<int> needed;

    for (int i = 0; i < 6; i++) {
        cin >> input[i];

        needed.push_back(pieceset[i] - input[i]); //take constants 
    }

    for (int i = 0; i < 6; i++) {
        cout << needed[i]  << " ";
    }

    return 0;
}

File: kattis/python/1_Easy/bijele.py
pieceset = [1, 1, 2, 2, 2, 8]

input_pieces = list(map(int, input().split()))

needed = []

for i in range(6):
    needed.append(pieceset[i] - input_pieces[i])

for i in range(6):
    print(needed[i], end=" ")
